ui/StatusQ/src/StatusQ/Controls/StatusSeedPhraseInput.qml

```diff
@@ -206,7 +206,7 @@ Item {
     Popup {
         id: suggListContainer
         contentWidth: seedSuggestionsList.width
-        contentHeight: ((seedSuggestionsList.count <= 5) ? seedSuggestionsList.count : 5) *34 + 16
+        contentHeight: ((seedSuggestionsList.count <= 5) ? seedSuggestionsList.count : 5) *34
```

Comment:
WDYT of extracting the delegate height in a variable since we need it here as well? 
Maybe also make the intention clear with an expression like:
Math.min(maxPopupHeight, seedSuggestionsList.contentHeight)